db/schema.rb

```diff
@@ -77,7 +77,7 @@
     t.string "instructors", default: [], null: false, array: true
     t.integer "conflicts", default: [], null: false, array: true
     t.uuid "uuid", null: false
-    t.string "periods_location", default: [], array: true
+    t.string "periods_location", default: [], null: false, array: true
```

Comment:
not sure how this ended up in the diff

Reply:
So would line 80 need to revert to original version?

Reply:
yes